### ISSUE TITLE
[BUG] Not specifying STORAGE_XYZ env vars causes HTTP 500 server error

### ISSUE THREAD

**Issue by michael-robbins** [AmruthPillai/Reactive-Resume#920]
**Describe the bug**
When you do not specify the `STORAGE_{BUCKET|REGION|etc}` env vars and attempt to upload a photo the server throws a HTTP 500 error and the client does not provide any error message, it just silently ignores the error and the user is left confused.

Server log:
```
[Nest] 42  - 06/18/2022, 9:16:33 PM   ERROR [ExceptionsHandler] Region is missing
Error: Region is missing
    at default (/app/node_modules/.pnpm/@aws-sdk+config-resolver@3.80.0/node_modules/@aws-sdk/config-resolver/dist-cjs/regionConfig/config.js:10:15)
    at /app/node_modules/.pnpm/@aws-sdk+node-config-provider@3.80.0/node_modules/@aws-sdk/node-config-provider/dist-cjs/fromStatic.js:6:83
    at /app/node_modules/.pnpm/@aws-sdk+property-provider@3.78.0/node_modules/@aws-sdk/property-provider/dist-cjs/chain.js:11:28
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async coalesceProvider (/app/node_modules/.pnpm/@aws-sdk+property-provider@3.78.0/node_modules/@aws-sdk/property-provider/dist-cjs/memoize.js:14:24)
    at async /app/node_modules/.pnpm/@aws-sdk+property-provider@3.78.0/node_modules/@aws-sdk/property-provider/dist-cjs/memoize.js:26:28
    at async Object.region (/app/node_modules/.pnpm/@aws-sdk+config-resolver@3.80.0/node_modules/@aws-sdk/config-resolver/dist-cjs/regionConfig/resolveRegionConfig.js:17:36)
    at async getEndpointFromRegion (/app/node_modules/.pnpm/@aws-sdk+config-resolver@3.80.0/node_modules/@aws-sdk/config-resolver/dist-cjs/endpointsConfig/utils/getEndpointFromRegion.js:7:20)
    at async serializeAws_restXmlPutObjectCommand (/app/node_modules/.pnpm/@aws-sdk+client-s3@3.81.0/node_modules/@aws-sdk/client-s3/dist-cjs/protocols/Aws_restXml.js:3525:68)
    at async /app/node_modules/.pnpm/@aws-sdk+middleware-serde@3.78.0/node_modules/@aws-sdk/middleware-serde/dist-cjs/serializerMiddleware.js:5:21
    at async /app/node_modules/.pnpm/@aws-sdk+middleware-logger@3.78.0/node_modules/@aws-sdk/middleware-logger/dist-cjs/loggerMiddleware.js:6:22
    at async ResumeService.uploadPhoto (/app/server/dist/resume/resume.service.js:164:9)
```

**Product Flavor**
- [ ] Managed (https://rxresu.me)
- [X] Self Hosted

**To Reproduce**
1. Start the self hosted container
2. Do *not* specify any `STORAGE_` env vars
3. Attempt to upload a profile photo image
4. Look at the container logs see error from above
5. Look at Dev Tools in browser and see HTTP 500 thrown and JS errors

**Expected behavior**
User is presented an error message saying something like 'profile photo upload failed, please try later or contact the admin'
Server logs something like: `'STORAGE_XYZ' env vars missing, disabled photo upload, returning client failure` or something like that :)

**Desktop (please complete the following information):**
Firefox browser, but shouldn't matter really.

**Comment by michael-robbins** [comment]
Looks like this will be closed with #906

**Comment by michael-robbins** [comment]
There's other issues tracking this so will close off this to have less clutter :)
